packages/appeals-service-api/__tests__/unit/routes/users.test.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const { routerMock } = require('../../routerMock');
+
+routerMock(express);
```

Comment:
Not sure we're actually testing anything here apart form that express router works properly? Apologies if I'm missing part of the context

Reply:
This is inline with the route tests on the front end